libsolidity/codegen/ContractCompiler.cpp

```diff
@@ -575,7 +575,7 @@ bool ContractCompiler::visit(InlineAssembly const& _inlineAssembly)
 			return true;
 		}
 	);
-	solAssert(errors.empty(), "Code generation for inline assembly with errors requested.");
+	solAssert(errors.empty() || Error::containsOnlyWarnings(errors), "Code generation for inline assembly with errors requested.");
```

Comment:
Then we should find a better name for that function `containsNoFatalErrors`? But yeah, different story.